.github/workflows/pr.yaml

```diff
@@ -16,6 +17,7 @@ jobs:
         working-directory: ./src/github.com/${{ github.repository }}
     env:
       GOPATH: ${{ github.workspace }}
+      KONK_NAMESPACE: konk
```

Comment:
call me paranoid, but I'd rather we test konk running in any namespace other than default or `konk`. This will separate any unintentional dependencies on resources in the operator namespace.

Moving the konk-operator to a different namespace would test the same thing.

Reply:
Moved to `baz`.